webapp/channels/src/components/edit_post/edit_post.tsx

```diff
@@ -99,6 +99,7 @@ const EditPost = ({editingPost, actions, canEditPost, config, channelId, draft,
         draft.message || editingPost?.post?.message_source || editingPost?.post?.message || '',
     );
     const [selectionRange, setSelectionRange] = useState<State['selectionRange']>({start: editText.length, end: editText.length});
+    const [caretPosition, setCaretPosition] = useState<number>(editText.length);
```

Comment:
Could this be stored in a ref instead? `selectionRange` probably could as well, but it's more important for `caretPosition` because we're setting it on every key press